shell/platform/darwin/macos/framework/Source/FlutterEngine.mm

```diff
@@ -603,6 +654,48 @@ - (FlutterPlatformViewController*)platformViewController {
 
 #pragma mark - Private methods
 
+- (FlutterEngineViewRecord*)viewRecordForId:(uint64_t)viewId {
+  if (viewId == kFlutterDefaultViewId) {
+    return _viewRecord;
+  }
+  return nil;
```

Comment:
Consider adding an assertion here. Currently it's unexpected to get the non-default view ID.